lib/paginate_firestore.dart

```diff
@@ -222,6 +225,22 @@ class _PaginateFirestoreState extends State<PaginateFirestore> {
       physics: widget.physics,
       slivers: [
         if (widget.header != null) widget.header!,
+        if (widget.headerBuilder != null)
+          SliverList(
+            delegate: SliverChildListDelegate(
+              [widget.headerBuilder!(context, loadedState.documentSnapshots)],
+            ),
+          ),
+        SliverPadding(
+          padding: widget.padding,
+          sliver: SliverList(
+            delegate: SliverChildBuilderDelegate(
+              (context, index) {
+                return;
+              },
+            ),
+          ),
+        ),
```

Comment:
Can you example why we need this part?

Reply:
>  I am developing an app named roadid ( roadid.com ), In this app there is a history section which has a header ( summary of the history records , like total activities, duration, user since, total distance etc ) which is an aggregate of the history data, that's why sometimes we need it ( to view a summarized data of a data set _

But I still don't understand why we need an empty `SliverChildBuilderDelegate` here.